code/modules/jobs/job_types/roguetown/adventurer/types/migrants/heartfelt/heartfeltlord.dm

```diff
@@ -77,6 +82,221 @@
 	ADD_TRAIT(H, TRAIT_NOBLE, TRAIT_GENERIC)
 	ADD_TRAIT(H, TRAIT_HEAVYARMOR, TRAIT_GENERIC)
 
+/datum/advclass/heartfelt/lord/archmage
+	name = "Archmagos of Heartfelt"
+	tutorial = "You are the Archmagos of Heartfelt, ruler of a once-prosperous acryne-barony now in ruin. \
+	Guided by visions of the great beyond, you journey to the Reach, seeking aid to restore your domain to its former glory, or perhaps claim a new throne."
+	category_tags = list(CTAG_HFT_LORD)
+	maximum_possible_slots = 1
+	outfit = /datum/outfit/job/heartfelt/lord/archmage
+	pickprob = 100
+	class_select_category = CLASS_CAT_HFT_COURT
+	subclass_social_rank = SOCIAL_RANK_NOBLE
+	traits_applied = list(TRAIT_MAGEARMOR, TRAIT_ARCYNE_T3, TRAIT_INTELLECTUAL, TRAIT_HEARTFELT)
+
+/datum/outfit/job/heartfelt/lord/archmage/pre_equip(mob/living/carbon/human/H)
+	..()
+	shirt = /obj/item/clothing/suit/roguetown/armor/gambeson/heavy
+	belt = /obj/item/storage/belt/rogue/leather/black
+	shoes = /obj/item/clothing/shoes/roguetown/boots/nobleboot
+	pants = /obj/item/clothing/under/roguetown/tights/black
+	cloak = /obj/item/clothing/cloak/heartfelt
+	armor = /obj/item/clothing/cloak/poncho/fancycoat
+	neck = /obj/item/storage/belt/rogue/pouch/coins/rich
+	beltl = /obj/item/flashlight/flare/torch/lantern
+	r_hand = /obj/item/rogueweapon/woodstaff/diamond
+	beltr = /obj/item/rogueweapon/huntingknife
+	gloves = /obj/item/clothing/gloves/roguetown/leather/black
+	backl = /obj/item/storage/backpack/rogue/satchel/heartfelt // Paper and Feather
+	backpack_contents = list(
+		/obj/item/recipe_book/alchemy,
+		/obj/item/roguegem/amethyst,
+		/obj/item/spellbook_unfinished/pre_arcyne,
+		/obj/item/rogueweapon/huntingknife/idagger/silver/arcyne,
+	)
+	id = /obj/item/scomstone
+	if(H.mind)
+		H.mind.AddSpell(new /obj/effect/proc_holder/spell/self/convertrole/heartfelt)
+	H.adjust_skillrank(/datum/skill/combat/polearms, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/magic/arcane, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/wrestling, 2, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/unarmed, 2, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/swimming, 2, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/climbing, 2, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/swords, 2, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/reading, 5, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/sneaking, 2, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/riding, 3, TRUE)
+	H.change_stat("intelligence", 3)
+	H.change_stat("endurance", 1)
+	H.change_stat("speed", 1)
+	H.change_stat("perception", 2)
+	H.change_stat("fortune", 5)
+	ADD_TRAIT(H, TRAIT_NOBLE, TRAIT_GENERIC)
+	ADD_TRAIT(H, TRAIT_ARCYNE_T3, TRAIT_GENERIC)
+	ADD_TRAIT(H, TRAIT_MAGEARMOR, TRAIT_GENERIC)
+	ADD_TRAIT(H, TRAIT_HEARTFELT, TRAIT_GENERIC)
+	if(H.mind)
+		H?.mind.adjust_spellpoints(24)
```

Comment:
Should probably be 3 points lower, you're already giving them a diamond staff at roundstart.

Reply:
The reasoning is that most full casters get 18-21 spell points, not 24, and they do not get a high-quality roundstart staff either.